pallets/ocex/src/lib.rs

```diff
@@ -1193,6 +1214,48 @@ pub mod pallet {
 			Self::deposit_event(Event::WithdrawFromOrderbook(user, asset, amount));
 			Ok(())
 		}
+
+		fn create_withdrawal_tree(
```

Comment:
Can we create an issue for this,  to refactor the snapshot summary to `BtreeMap` on the client side or are we agreeing to use this in the future as well?

Reply:
done